deprecated/fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourcePackActivationType.java

```diff
@@ -16,22 +16,31 @@
 
 package net.fabricmc.fabric.api.resource;
 
+import net.fabricmc.fabric.api.resource.v1.pack.PackActivationType;
+
 /**
  * Represents the resource pack activation type.
  */
+@Deprecated
 public enum ResourcePackActivationType {
 	/**
 	 * Normal activation. The user has full control over the activation of the resource pack.
 	 */
-	NORMAL,
+	NORMAL(PackActivationType.NORMAL),
 	/**
 	 * Enabled by default. The user has still full control over the activation of the resource pack.
 	 */
-	DEFAULT_ENABLED,
+	DEFAULT_ENABLED(PackActivationType.DEFAULT_ENABLED),
 	/**
 	 * Always enabled. The user cannot disable the resource pack.
 	 */
-	ALWAYS_ENABLED;
+	ALWAYS_ENABLED(PackActivationType.ALWAYS_ENABLED),;
```

Comment:
Nit: the `,;` formatting is a bit strange. I don't think FAPI does it elsewhere.